Traceback (most recent call last):
  File "/home/dsemchin/Progression_models_simulations/scripts/run_mcmc_inference.py", line 167, in <module>
    config_file = sys.argv[1]
                  ~~~~~~~~^^^
IndexError: list index out of range
